Unable to build firmware

**Problem Description**
I was following the instructions, but was unable to build the firmware through Arduino IDE (1.8.2)

The sketchbook is set as `~/catkin_ws/src/versavis/firmware`

The error log is as the following. 
```
Arduino: 1.8.2 (Linux), Board: "VersaVIS"

Build options changed, rebuilding all
In file included from /home/jaeyoung/catkin_ws/src/versavis/firmware/libraries/versavis/src/Imu.h:17:0,
                 from /home/jaeyoung/catkin_ws/src/versavis/firmware/libraries/versavis/src/ADIS16448BMLZ.h:33,
                 from /home/jaeyoung/catkin_ws/src/versavis/firmware/versavis/versavis.ino:15:
/home/jaeyoung/catkin_ws/src/versavis/firmware/libraries/versavis/src/Sensor.h:19:10: fatal error: versavis/ImuMicro.h: No such file or directory
 #include <versavis/ImuMicro.h>
          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
exit status 1
Error compiling for board VersaVIS.

This report would have more information with
"Show verbose output during compilation"
option enabled in File -> Preferences.

```
`ImuMicro.h` seems to be something that is built afterwards and added into the `catkin_ws/devel/versavis`, but arduinoIDE is unable to find it

--- a/firmware/setup.sh
+++ b/firmware/setup.sh
@@ -1,5 +1,5 @@
 catkin build versavis
-shell=`grep ^$(id -un): /etc/passwd | cut -d : -f 7- |  cut -d'/' -f4-`
+shell=`echo $0 | cut -c 2-`
 devel=/`catkin config | grep ^"Devel Space:" |  cut -d'/' -f2-`
 source_path="$devel"/setup."$shell"
 

--- a/firmware/setup.sh
+++ b/firmware/setup.sh
@@ -1,7 +1,8 @@
+#!/bin/bash
+
 catkin build versavis
-shell=`echo $0 | cut -c 2-`
 devel=/`catkin config | grep ^"Devel Space:" |  cut -d'/' -f2-`
-source_path="$devel"/setup."$shell"
+source_path="$devel"/setup.bash
 
 echo $source_path
 source $source_path

